system/security-critical-code.md

```diff
@@ -40,19 +44,14 @@ These are the components that we haven't written or designed ourselves, yet we s
 -   The rpm program used in Dom0 for verifying Dom0 updates' signatures
 -   Somehow optional: log viewing software in dom0 that parses VM-influenced logs
 
-Additionally when we consider attacks that originate through a compromised network domain and target the VMs connected to it. Those attacks do not apply to domains connected to other network domains (Qubes allows more than one network domains), or those with networking disabled (Dom0 is not connected to any network by default).
+Additionally when we consider attacks that originate through a compromised network domain and target the varying VMs connected to it, these parts must also be included. Those attacks do not apply to domains connected to other network domains (Qubes allows more than one network domains), or those with networking disabled (Dom0 is not connected to any network by default).
 
 -   Xen network PV frontends
 -   VM's core networking stacks (core TCP/IP code)
 
-Buggy Code vs. Back-doored Code Distinction
+On "Non-Security-Critical" Components
 -------------------------------------------
 
-There is an important distinction between the buggy code and maliciously trojaned code. We could have the most secure architecture and the most bulletproof TCB that perfectly isolates all domains from each other, but it still would be pretty useless if all the code used within domains, e.g. the actual email clients, word processors, etc, was somehow trojaned. In that case only network-isolated domains could be somehow trusted, while all others could be not.
+We could have finely audited architecture and a bulletproof TCB that perfectly isolates all domains from each other, but that security would be wasted if the actual email clients, word processors, and other apps used with Qubes OS were somehow trojaned. Treating the *security critical components* as the *only necessary components* means trusting solely the network-isolated domains.
```

Comment:
Really? I vehemently disagree. 

> There is an important distinction between the buggy code and maliciously Trojaned code.
The paragraph does not explain that distinction at all, so I removed the sentence.

> We could have the most secure architecture and the most bulletproof TCB 
"the most secure architecture and the most bulletproof TCB" is awkward phrasing that sounds somewhat childish. 

> but it still would be pretty useless if all the code used within domains, e.g. the actual email clients, word processors, etc, was somehow trojaned.
Again, "still would be pretty useless" and an overuse of eg., etc., in the article end up making it sound weird, plus the subject-verb-object agreement issue.
> In that case only network-isolated domains could be somehow trusted, while all others could be not.
This just needed clarification and I think the way I phrased is much more impactful.

That said, it's subjective, so if you want me to drop it I'll trim it down to just the grammar changes and resubmit. I think this is magnitudes better than what it was earlier though.